src/Framework/Framework.depproj

```diff
@@ -16,15 +16,11 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
     </ProjectReference>
-    <ProjectReference Include="Framework-native.depproj" Condition="$(TargetsUnix) and '$(Platform)' != 'wasm'">
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-      <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
-    </ProjectReference>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Private.CoreFx.NETCoreApp">
-      <Version>$(CoreFxVersion)</Version>
+    <PackageReference Include="Microsoft.NETCore.App.Runtime.$(NuPkgRid)">
```

Comment:
Unlike the previous package, this one does not have .pdbs and I am not able to figure out where and how to get them

Reply:
I think that's not a big deal for now since we don't have SourceLink anyway.

I'll try to have a look whether we can at least generate debug info for method parameters in the absence of PDBs (IIRC we don't generate any debug info right now if there's no ILPDB).